### ISSUE TITLE
twisted.internet.udp.Port.write_sequence needs a unit test

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8644]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8644|
|Type|enhancement|
|Created|2016-07-21 08:53:07Z|
|Branch|https://github.com/twisted/twisted/tree/8644-rodrigc-write_sequence-test|

twisted.internet.udp.Port.write_sequence needs a unit test

<details><summary>Searchable metadata</summary>

```
trac-id__8644 8644
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__None None
branch__8644_rodrigc_write_sequence_test 8644-rodrigc-write_sequence-test
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1469091187286021 1469091187286021
changetime__1469142441986746 1469142441986746
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

https://github.com/twisted/twisted/pull/378

**Comment by twisted-trac** [comment]
|[<img alt="scop's avatar" src="https://avatars.githubusercontent.com/u/109152?s=50" width="50" height="50">](https://github.com/scop)<a name="note_2"></a>|@scop set owner to @rodrigc|
|-|-|

LGTM, please merge

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|GitHub set status to `closed`|
|-|-|

In changeset 71f7b90c4905d984fc46aca7f455fbd6113607bd
```
#!CommitTicketReference repository="" revision="71f7b90c4905d984fc46aca7f455fbd6113607bd"
Merge pull request #378 from twisted/8644-rodrigc-write_sequence-test

Author: rodrigc
Reviewer: scop
Fixes: #8644

Add test for twisted.internet.udp.Port.writeSequence
```
